lib/components/ActionSheet/footer/index.tsx

```diff
@@ -1,10 +1,11 @@
 import { FooterProps } from "../types";
+import { useContext } from "react";
```

Comment:
Just reverting ActionSheet changes in this file from this [PR](https://github.com/deriv-com/quill-ui/pull/303/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)